### ISSUE TITLE
Remove layout_array and telescope positions warning

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/gammasim-tools#307]
Every initialization of ArraySimulator or ShowerSimulator results in a couple of lines of warnings related to layout_array and telescope_positions. Very likely this is not useful / relevant.

e.g., 
```
WARNING::layout_array(l578)::_getCrsLocal::crs_local cannot be built because center lon and lat are missing
WARNING::layout_array(l590)::_getCrsUtm::crs_utm cannot be built because center lon and lat are missing
WARNING::telescope_position(l612)::convertAll::crsLocal is None - conversions will be impacted
WARNING::telescope_position(l614)::convertAll::crsUtm is None - conversions will be impacted
WARNING::telescope_position(l650)::convertAll::Warning: telescope height might be incorrect du to incomplete CORSIKA observation (None)or sphere centre information (None)
INFO::layout_array(l298)::readTelescopeListFile::Reading telescope list from /afs/ifh.de/group/cta/scratch/maierg/CTA/MC-Pipeline/gammasim-tools/data/layout/telescope_positions-North-Prod5.ecsv
INFO::corsika_config(l499)::_setOutputFileAndDirectory::Creating directory /afs/ifh.de/group/cta/scratch/maierg/CTA/MC-Pipeline/gammasim-tools/simtools-output/test-production/corsika, if needed.
WARNING::layout_array(l578)::_getCrsLocal::crs_local cannot be built because center lon and lat are missing
WARNING::layout_array(l590)::_getCrsUtm::crs_utm cannot be built because center lon and lat are missing
WARNING::telescope_position(l612)::convertAll::crsLocal is None - conversions will be impacted
WARNING::telescope_position(l614)::convertAll::crsUtm is None - conversions will be impacted
WARNING::telescope_position(l650)::convertAll::Warning: telescope height might be incorrect du to incomplete CORSIKA observation (None)or sphere centre information (None)
INFO::layout_array(l298)::readTelescopeListFile::Reading telescope list from /afs/ifh.de/group/cta/scratch/maierg/CTA/MC-Pipeline/gammasim-tools/data/layout/telescope_positions-North-Prod5.ecsv
```

**Comment by orelgueta** [comment]
Does #323 mean this is resolved and can be closed?

**Comment by GernotMaier** [comment]
Fixed with #323
